workflows/src/test/kotlin/com/r3/corda/lib/tokens/workflows/CreateEvolvableTokenTests.kt

```diff
@@ -165,4 +166,65 @@ class CreateEvolvableTokenTests : JITMockNetworkTests() {
             alice.createEvolvableToken(token, notaryIdentity).getOrThrow()
         }
     }
+
+    // Flows with observers
+    @Test
+    fun `with unrelated observer`() {
+        // denise as observer
+        val token = factory.withOneMaintainerAndOneParticipant()
+
+        val createTx = alice.createEvolvableToken(token, notaryIdentity, listOf(denise.legalIdentity())).getOrThrow()
+        val createdToken = createTx.singleOutput<TestEvolvableTokenType>()
+
+        // Expect to have one create command with maintainer signature
+        val maintainerKeys = token.maintainers.map { it.owningKey }.toSet()
+        assertEquals(maintainerKeys, createTx.requiredSigningKeys, "Must be signed by all maintainers")
+
+        // Alice, Charlie, Denise should record the transaction
+        assertHasTransaction(createTx, alice, charlie, denise)
+        val aliceToken = alice.services.vaultService.queryBy<TestEvolvableTokenType>().states
```

Comment:
I assume alice is the issuer here and the other two are the observer / participant?

Reply:
I added comments.

Reply:
👍